ports/esp32s2/common-hal/analogio/AnalogIn.c

```diff
@@ -34,6 +34,8 @@
 
 #include "shared-bindings/microcontroller/Pin.h"
 
+#include <string.h>
+
 #define DEFAULT_VREF        1100
 #define NO_OF_SAMPLES       64
```

Comment:
I discussed this choice of averaging 64 samples with @ladyada, and we thought we could reduce this to 2 instead of 64. I did some instrumentation and all 64 values were fairly close; there's not a lot of need to average so many samples.

Reply:
Ok, I'll swap that out